### PR TITLE
[EPMRPP-39882] Last login for ldap

### PR DESCRIPTION
**Pull request by ihar-kahadouski**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -46,9 +46,9 @@ dependencies {
         compile 'com.epam.reportportal:commons-rules:5.0.0-BETA-6'
         compile 'com.epam.reportportal:commons-model:5.0.0-BETA-8'
     } else {
-        compile 'com.github.reportportal:commons-dao:5.0.0-BETA-7'
+        compile 'com.github.reportportal:commons-dao:9b640a28'
```
Comment (pbortnik) [inline build.gradle]: if there were no changes, please return back the release version

File: build.gradle [hunk 2]
```diff
@@ -46,9 +46,9 @@ dependencies {
...
         compile 'com.github.reportportal:commons-rules:5.0.0-BETA-6'
-        compile 'com.github.reportportal:commons-model:5.0.0-BETA-8'
+        compile 'com.github.reportportal:commons-model:9f72549'
```
Comment (pbortnik) [inline build.gradle]: if there were no changes, please return back the release version

Comment (ihar-kahadouski) [reply build.gradle]: done

File: build.gradle (hunk 1 above)
Comment (ihar-kahadouski) [reply build.gradle]: done

**Pull request merged**